e2e/examples.spec.ts

```diff
@@ -1,87 +1,45 @@
-  await page.getByPlaceholder('Password',  { exact: true }).fill(password || 'BBBB');
-  await page.getByRole('button', { name: 'Sign In' }).click();
-
-  await page.getByLabel('Security Data Lake Add-On for Splunk').click()
-
-  console.log("Create account: ", createAccount, ", set to true to create account")
-  if ( createAccount == 'true') {
-    // Go to configuration
-    await page.goto('/en-GB/app/TA_dataset/configuration');
-
-    // Check that we are on the configuration page
-    await expect(page).toHaveTitle(/Configuration/);
-
-    // Open dialog
-    const locAddDialog = page.locator('#accountTab').getByRole('button', { name: 'Add' });
-    await locAddDialog.click();
-
-    // Setup locators
-    const locAccount = page.locator('div').filter({ hasText: /^Account nameEnter a unique name for this account\.$/ }).locator('[data-test="textbox"]');
-    const locUrl = page.locator('div').filter({ hasText: /^URLEnter DataSet URL\.$/ }).locator('[data-test="textbox"]');
-    const locReadKey = page.locator('[data-test="body"] form div').filter({ hasText: 'DataSet Log Read Access KeyRequired to enable inputs and SPL comand. Include tra' }).locator('[data-test="textbox"]');
-    const locWriteKey = page.locator('[data-test="body"] form div').filter({ hasText: 'DataSet Log Write Access KeyRequired to enable alert action. Include trailing hy' }).locator('[data-test="textbox"]');
-
-    // Read env with values
-    const { DATASET_URL: datasetUrl, DATASET_LOG_ACCESS_READ: datasetReadKey, DATASET_LOG_ACCESS_WRITE: datasetWriteKey } = process.env;
-
-    // Fill in values
-    const accountName = 'QaTr' + (Math.random() * 1e18)
-    await locAccount.fill(accountName);
-    await locUrl.fill(datasetUrl || '');
-    await locReadKey.fill(datasetReadKey || '');
-    await locWriteKey.fill(datasetWriteKey || '');
 
-    // Click on Add
-    const locAddConfirm = page.locator('[data-test="footer"]').getByRole('button', { name: 'Add' })
-    await locAddConfirm.click();
-
-    // await page.getByLabel(accountName).click()
-  }
+  await goToDataSet(page);
 });
 
+
 test('Check example page', async ({ page }) => {
-  await page.goto('en-GB/app/TA_dataset/dataset_by_example');
+  console.log("Go to example page");
+  await page.locator('[title="DataSet by Example"]').click();
 
 
   await expect(page).toHaveTitle(/DataSet by Example/);
 
+  // wait for elements to load
+  await expect(page.getByRole("main").getByText("4. Timeseries Query: This will calculate numeric values over time.")).toHaveCount(1)
+
   // Wait for the "Waiting for data..." text to disappear
-  await page.waitForFunction(() => {
-    const element = document.querySelector('body');
-    return !element.textContent.includes('Waiting for data...');
-  });
+  console.log("Wait for 'Waiting for data' to disappear")
+  await page.screenshot({ path: 'playwright-screenshots/page-examples-before-waiting-for-data.png', fullPage: true });
```

Comment:
I wonder about the screenshots. It's not common to just take screenshots in tests. I think we should either take a screenshot of a failed test (this is what we do in dataset FE) or if it's just for development then run Playwright in the "ui" mode.